### PR TITLE
feat: add account nft template

### PR DESCRIPTION
**Pull request by jorgeantonio21**
Description
---
Creates a built-in template for accounts minting NFTs.

Motivation and Context
---

How Has This Been Tested?
---

What process can a PR reviewer use to test or verify this change?
---

Breaking Changes
---

- [x] None
- [ ] Requires data directory to be deleted
- [ ] Other - Please specify

### CODE REVIEW

File: dan_layer/template_lib/src/models/metadata.rs [hunk 1]
```diff
@@ -58,3 +60,13 @@ impl Default for Metadata {
...
     }
 }
+
+impl Display for Metadata {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        let mut s = vec![];
+        for (key, value) in self.0.iter() {
+            s.push(format!("key = {}, value = {}", key, value))
+        }
+        write!(f, "Metadata: {:?}", s)
```
Comment (sdbondi) [inline dan_layer/template_lib/src/models/metadata.rs]: Generally, the implementation of Display should never allocate
```suggestion
        write!(f, "Metadata: ")
        for (key, value) in self.0.iter() {
            write!!("key = {}, value = {} ", key, value))
        }

```

File: dan_layer/template_lib/src/models/metadata.rs [hunk 2]
```diff
@@ -20,6 +20,8 @@
 //   WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE
 //   USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
+use std::fmt::Display;
```
Comment (sdbondi) [inline dan_layer/template_lib/src/models/metadata.rs]: no_std support
```suggestion
use tari_template_abi::rust::fmt::Display;
```

**Review by sdbondi — DISMISSED** [review]
Looks good

File: dan_layer/template_lib/src/models/metadata.rs (hunk 1 above)
Comment (jorgeantonio21) [reply dan_layer/template_lib/src/models/metadata.rs]: Thanks, addressed !

**Review by sdbondi — APPROVED** [review]

**Pull request merged**